examples/basic/packages/ui/Header.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from "react";
+
+export const Header = ({ text }: { text: string }) => {
+  return <h1>{text}aaaaa</h1>;
```

Comment:
something better here?

Reply:
Ha, forgot I left that there when I was debugging that caching question.